tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/Tcl.xtext

```diff
@@ -51,6 +49,22 @@ TestStep:
 	'-' contents+=StepContent* '.'?
 ;
 
+TestStepAssertion:
+	'-' 'assert' (negated?='!')? variableName=ID (comparator=Comparator contents+=StepContent)? '.'?
```

Comment:
I would see the `!` as part of the not-null operator only - or some future more powerful expressions like `!(...)`. Right now, it doesn't make sense to write `!x == "something"` and might be confusing. Does it mean `!(x == "something")` or does it mean `(!x) == "something"`?